src/polymer/polymer-element.ts

```diff
@@ -221,14 +221,15 @@ function _getFlattenedAndResolvedBehaviors(
     //     array, which should be the most recently defined one.
     const foundBehavior = document.getOnlyAtId('behavior', behavior.name);
     if (!foundBehavior) {
-      throw new WarningCarryingException({
+      document.warnings.push({
```

Comment:
Test added, but I just remembered this was already solved for during the scanning phase (by saving the proper sourceRange to be used here) instead of during this detection phase.  So while a warning is generated because the behavior wasn't detected for the `index.html` document, it is generated with the `sourceRange` cached in the `ScannedBehaviorAssignment` pointing to where the behavior assignment was referenced.